bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPSiteQuality.java

```diff
@@ -112,46 +113,52 @@ public String toString() {
         public TimingWindow clone() {
             try {
                 return (TimingWindow) super.clone();
-            } catch (CloneNotSupportedException e) {
+            } catch (final CloneNotSupportedException e) {
                 throw new Error("This was supposed to be impossible.");
             }
         }
 
+        @Override
+        public boolean equals(final Object other) {
```

Comment:
I needed to add an `equals` method to `TimingWindow` for the `TimingWindowImporterSpec` comparisons to work properly.